src/System.CommandLine.Tests/DirectiveTests.cs

```diff
@@ -100,63 +95,67 @@ public void Directives_must_have_a_non_empty_key(string directive)
 
             var result = option.Parse($"{directive} -a");
 
-            result.Directives.Should().BeEmpty();
             result.UnmatchedTokens.Should().Contain(directive);
         }
 
         [Theory]
         [InlineData("[par se]")]
         [InlineData("[ parse]")]
         [InlineData("[parse ]")]
-        public void Directives_cannot_contain_spaces(object value)
+        public void Directives_cannot_contain_spaces(string value)
         {
-            var option = new Option<bool>("-a");
-
-            var result = option.Parse($"{value} -a");
+            Action create = () => new Directive(value);
 
-            result.Directives.Should().BeEmpty();
+            create.Should().Throw<ArgumentException>();
         }
 
-        [Fact]
-        public void When_a_directive_is_specified_more_than_once_then_its_values_are_aggregated()
-        {
-            var option = new Option<bool>("-a");
+        //[Fact]
+        //public void When_a_directive_is_specified_more_than_once_then_its_values_are_aggregated()
+        //{
+        //    var option = new Option<bool>("-a");
 
-            var result = option.Parse("[directive:one] [directive:two] -a");
+        //    var result = option.Parse("[directive:one] [directive:two] -a");
 
-            result.Directives.TryGetValue("directive", out var values).Should().BeTrue();
-            values.Should().BeEquivalentTo("one", "two");
-        }
+        //    result.Directives.TryGetValue("directive", out var values).Should().BeTrue();
+        //    values.Should().BeEquivalentTo("one", "two");
+        //}
 
-        [Fact]
-        public void Directive_count_is_based_on_distinct_instances_of_directive_name()
-        {
-            var command = new RootCommand();
+        //[Fact]
+        //public void Directive_count_is_based_on_distinct_instances_of_directive_name()
+        //{
+        //    var command = new RootCommand();
 
-            var result = command.Parse("[one] [two] [one:a] [one:b]");
+        //    var result = command.Parse("[one] [two] [one:a] [one:b]");
 
-            result.Directives.Should().HaveCount(2);
-        }
+        //    result.Directives.Should().HaveCount(2);
+        //}
 
         [Fact]
-        public void Directives_can_be_disabled()
+        public void When_directives_are_not_enabled_they_are_treated_as_regular_tokens()
```

Comment:
I would expect them to be present as tokens now in any case, though maybe only on the root command result.